#Message type for perch detections
Header header
geometry_msgs/Point axis_point
geometry_msgs/Point axis_direction
float32 radius
float32 fitting_score